client/src/pages/Lobby/Lobby.tsx

```diff
@@ -1,49 +1,66 @@
 import React, { FC, useEffect, useState } from 'react'
-import axios from 'axios'
+import axios, { AxiosResponse } from 'axios'
 import { RouteComponentProps } from '@reach/router'
 import { Game } from '../../types/Game'
 import * as moment from 'moment'
 
+import connectToGameHub from '../../utils/signalrConnector'
+
 import './Lobby.scss'
+import PlayerBox from '../../components/PlayerBox'
 
 export interface LobbyProps extends RouteComponentProps {
   id?: string
 }
 
 const Lobby: FC<LobbyProps> = (props: LobbyProps) => {
   const [game, setGame] = useState<Game>()
-  
+  const [currentPlayerId, setCurrentPlayerId] = useState<string>()
+
   useEffect(() => {
```

Comment:
All this could be made into a hook like `useLobby` or `useGame` or something. 

```es6
const { game, playerId } = useGame(props.id)
```

Reply:
Let's keep things simple for now, and consider creating a custom hook once the code is more stable.